source/includes/_organizations.md

```diff
@@ -77,3 +77,187 @@ Get a list of active job postings for a company.
+            "display_name": "Apollo.io"
+        },
+        {
+            "label": "# Employees",
+            "signal_field_name": "organization_num_employees_ranges",
+            "value": "1,100",
+            "display_name": "1-100"
+        },
+        {
+            "label": "# Employees",
+            "signal_field_name": "organization_num_employees_ranges",
+            "value": "1,1000",
+            "display_name": "1-1000"
+        },
+        {
+            "label": "Company Locations",
+            "signal_field_name": "organization_locations",
+            "value": "United States",
+            "display_name": "United States"
+        },
+        {
+            "label": "Company Keywords Contain ANY Of",
+            "signal_field_name": "q_organization_keyword_tags",
+            "value": "sales strategy",
+            "display_name": "sales strategy"
+        },
+        {
+            "label": "Company Keywords Contain ANY Of",
+            "signal_field_name": "q_organization_keyword_tags",
+            "value": "lead",
+            "display_name": "lead"
+        }
+    ],
+    "partial_results_only": false,
+    "disable_eu_prospecting": false,
+    "partial_results_limit": 10000,
+    "pagination": {
+        "page": 1,
+        "per_page": 10,
+        "total_entries": 1,
+        "total_pages": 1
+    },
+    "accounts": [
+        {
+            "id": "63b3e47f0deb820001f19XXX",
+            "name": "Apollo.io",
+            "website_url": null,
+            "blog_url": null,
+            "angellist_url": null,
+            "linkedin_url": "http://www.linkedin.com/company/apollxxx",
+            "twitter_url": "https://twitter.com/MeetApoxxx/",
+            "facebook_url": "https://facebook.com/MeetApoxxx/",
+            "primary_phone": {
+                "number": "+1 415-640-9303",
+                "source": "Account"
+            },
+            "languages": [],
+            "alexa_ranking": 3514,
+            "phone": null,
+            "linkedin_uid": "18511550",
+            "founded_year": 2015,
+            "publicly_traded_symbol": null,
+            "publicly_traded_exchange": null,
+            "logo_url": null,
+            "crunchbase_url": null,
+            "primary_domain": null,
+            "sanitized_phone": "+14156409303",
+            "owned_by_organization_id": null,
+            "organization_raw_address": "535 mission st, suite 1100, san francisco, california 94105, us",
+            "organization_city": "San Francisco",
+            "organization_street_address": "535 Mission St",
+            "organization_state": "California",
+            "organization_country": "United States",
+            "organization_postal_code": "94105",
+            "suggest_location_enrichment": false,
+            "parent_account": {
+                "id": "62b5283531e8210001210XXX",
+                "name": "Apollo.io"
+            },
+            "domain": "apollo.io",
+            ...
```

Comment:
Is this expected ?

Reply:
we should get this confirmed from product.

Reply:
I think documentation should have all the fields as it helps someone to integrate that in their systems

Reply:
Removed the` ...` and added all the fields.

Reply:
Checked across other places in documentation, we are not essentially removing the fields, just reducing the values if the response is too large. Made consistent with that behaviour.